core/src/tasks/settle_match.rs

```diff
@@ -49,9 +49,12 @@ use crate::{
         orderbook_management::{OrderBookManagementMessage, ORDER_BOOK_TOPIC},
```

Comment:
The `tasks` crate is essentially all going to be deleted in the next step of the refactor, changed this specific interface to allow refactoring of the `handshake` crate.